### ISSUE TITLE
[BUG] <title>lora微调，相同参数配置，达到相同收敛的loss(如0.0002)，单张A100比2张A100速度快，请问是什么原因呢？

### ISSUE THREAD

**Issue by sunyclj** [QwenLM/Qwen#669]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

### 备注 | Anything else?

测试如下：
单GPU:{'loss': 0.0002, 'learning_rate': 0.00021013958031099205, 'epoch': 50.0}
耗时：103.4079s
双GPU:{'loss': 0.0002, 'learning_rate': 0.0003, 'epoch': 155.0}
耗时：181.8167s

**Comment by jklj077** [comment]
如果用本repo的脚本，固定的是per_device_train_batch_size。多卡微调没改这个参数的话，实际batchsize就翻了n倍。您确认下这边配置是对齐的吗？
